src/components/SongCard/SongCard.css

```diff
@@ -28,8 +28,19 @@
   width: 100%;
 }
 
-.add-btn {
+/* don't see where this is getting applied? */
+/* .add-btn {
   position: absolute;
   right: 5px;
   top: 50%;
+} */
```

Comment:
Maybe you had branched off from an earlier point but this is where the add-btn positioning is defined

Reply:
ahh ok. let me merge with main and I'll check it out

Reply:
So I pulled in main but it said I was up to date. Where is the .add-btn class being used?

Reply:
I don't see it being used in the SongCard component which is where I remember it being but I can re-apply it after taking in account the change to can icon